### DISCUSSION TITLE
portal接入OIDC，用户会话过期状态可以跟随jwt token的过期时间吗？

### DISCUSSION THREAD

**Discussion by a517363523** [apolloconfig/apollo#5292] [Q&A]
portal接入OIDC，用户会话过期状态可以跟随jwt token的过期时间吗？

**Comment by nobodyiam** [comment]
我理解这部分的具体逻辑是 OIDC 的实现控制的，可以实际测试一下

**Reply by a517363523** [reply]
不行，我试了好像只能通过spring的参数server.servlet.session.timeout 来设置，另外好像也不支持OIDC的RefreshToken功能
